lib/chat_api_web/controllers/slack_controller.ex

```diff
@@ -6,14 +6,8 @@ defmodule ChatApiWeb.SlackController do
   action_fallback ChatApiWeb.FallbackController
 
   def oauth(conn, %{"code" => code}) do
-    IO.inspect("Code from Slack OAuth:")
```

Comment:
can we leave these in for now? very useful for debugging slack integrations (until we've had a few people set up slack successfully, it's helpful to have these logs)